fibers.cpp

```diff
@@ -38,23 +38,27 @@
 #include "win.h"
 #include "qpc.h"
 #include <functional>
+#include "gcstring.h"
 
 //#include "ostreamcon.h"
 //#define LOG(stuff) con() << stuff << endl
 #define LOG(stuff)
 
+static int64 fiber_count = 0;

```

Comment:
I would say int64 is overkill. At one per second, an int will handle 68 years (Wolfram Alpha is great for this kind of calculation, just enter "2 ^ 31 seconds")